corehq/apps/userreports/specs.py

```diff
@@ -13,7 +13,8 @@ def TypeProperty(value):
 class EvaluationContext(object):
     """
     An evaluation context. Necessary for repeats to pass both the row of the repeat as well
-    as the root document.
+    as the root document and the iteration number.
     """
-    def __init__(self, root_doc):
+    def __init__(self, root_doc, iteration):
```

Comment:
is subjective, but if you made the default iteration number 0 that will make it less annoying to remember the API for future extensions that aren't concerned with repeats.